tests/TestKdbx4.cpp

```diff
@@ -147,6 +148,14 @@ void TestKdbx4::testFormat400Upgrade()
     // upgrade to KDBX 4 by changing KDF and Cipher
     sourceDb->changeKdf(KeePass2::uuidToKdf(kdfUuid));
     sourceDb->setCipher(cipherUuid);
+
+    if (addCustomData) {
+        QVariantMap publicCustomData;
+        publicCustomData.insert("CustomPublicData", "Hey look, I turned myself into a pickle!");
+        sourceDb->setPublicCustomData(publicCustomData);
+        sourceDb->rootGroup()->customData()->set("CustomGroupData", "I just killed my family! I don't care who they were!");
```

Comment:
A little morbid...

Reply:
It's a quote. :-D

Reply:
Gotcha, I don't watch Rick and Morty... although maybe I should!

Reply:
You should.